src/assets/scripts/client/App.js

```diff
@@ -84,7 +84,9 @@ export default class App {
             $.getJSON('assets/aircraft/aircraft.json')
         )
             .done((airportResponse, airlineResponse, aircraftResponse) => {
-                const initialAirport = typeof airportResponse[0] !== 'undefined' ? airportResponse[0] : null;
+                const initialAirport = typeof airportResponse[0] !== 'undefined' 
+                ? airportResponse[0] 
```

Comment:
so close!

the lines after the ternary conditional should be indented 4 spaces:
```javascript
const initialAirport = typeof airportResponse[0] !== 'undefined' 
    ? airportResponse[0] 
    : null;
```